client/tokenized-payment-request/cart-api.js

```diff
@@ -117,13 +125,14 @@ export default class PaymentRequestCartApi {
 	 *          billing_address: Object?,
 	 *          shipping_address: Object?,
 	 *        }} customerData Customer data to update.
+	 * @param {string} context The context in which the request is being made.
 	 * @return {Promise} Cart Response on success, or an Error Response on failure.
 	 */
-	async updateCustomer( customerData ) {
+	async updateCustomer( customerData, context ) {
 		return await this._request( {
 			method: 'POST',
 			path: '/wc/store/v1/cart/update-customer',
-			credentials: 'omit',
+			credentials: context === 'product' ? 'omit' : 'same-origin',
```

Comment:
Rather than passing an additional argument to these methods, what do you think about using the `getPaymentRequestData` utility - which is already used in this file?

```suggestion
			credentials: getPaymentRequestData( 'button_context' ) === 'product' ? 'omit' : undefined,
```

Reply:
Good call! [e8f655a](https://github.com/Automattic/woocommerce-payments/pull/8891/commits/e8f655a866e3aff1c19d9a1d952c21ee68a0d30f)